.github/workflows/build.yml

```diff
@@ -20,19 +20,23 @@ jobs:
           - os: macos-latest
             node-version: 10.x
     steps:
-      - uses: actions/checkout@v1
-      - name: Get yarn cache directory path
-        id: yarn-cache-dir-path
-        run: echo "::set-output name=dir::$(yarn cache dir)"
-      - uses: actions/setup-node@v1
+      - uses: actions/checkout@v2 
+      - run: git fetch --depth=1
+      - uses: actions/setup-node@v2
         with:
           node-version: ${{ matrix.node-version }}
       - uses: actions/cache@v2
         id: cache
         with:
-          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          path: |
+            node_modules
+            */*/node_modules
+            **/dist
           key: ${{ matrix.os }}-${{ hashFiles('**/yarn.lock') }}
       - run: yarn install --frozen-lockfile
+        if: steps.cache.outputs.cache-hit != 'true'
```

Comment:
minor: I would use `== 'false'`  or whit do you compare to != true?

Reply:
`steps.cache.outputs.cache-hit` returns a boolean so yes, false would be the cleaner approach you are correct, thanks for the feedback